Fix/multiple choice slider

boranfurkan opened the pull request:
- Description:
- ClickUp links:
- What are the acceptance criteria? What are the steps to test that this code is working?

In this pr I have resolved the following two problems:

1-) https://app.clickup.com/t/86byujq5g
2-) https://app.clickup.com/t/86byujqb6

I have used the custom primary slider component that I have created earlier in https://app.clickup.com/t/86byhc3wa. I have tried to make sure it did not affect the other places where updated component renders. Out of my checks I have spotted that it was being rendered in 2 places only. The first one is the one I meant to fix anyways. In the second place (see below):

<img width="1207" alt="image" src="https://github.com/gator-labs/chomp/assets/96897046/d815f4df-ea96-43ac-a45f-184a3ddf0d27">

I have realised that, we are trying to give some color hexs to slider. I have tried to find the design of this in the Figma but I could not. If @AnteVuletic  or @bdeak4 could tell me about this component or provides me the design in figma, I can also make sure for that, it matches with current. This is how the second order slider works at the moment after my updates:

https://github.com/gator-labs/chomp/assets/96897046/b71cefaf-afe3-4c3e-947e-fe3992b5045f

AnteVuletic commented:
If you're asking where the content is displayed it should be on the home feed. If you have a answered multiple choice question which is revealed this content should appear. If you're trying to find it on figma it's in the alpha screens. Sample screenshot:
![image](https://github.com/gator-labs/chomp/assets/43707413/8674e6d6-8862-4baa-9d54-30daf5c5fc23)

boranfurkan commented:
So two things to mention here: 

1-) I have resolved the problem with  AnswerResult component's to be rendered other places problem. I made sure it did not affect any other places. 

2-) After I updated this pr for the problem above, I have realised that it raised some conflict problems. After deeper look into that, I have seen that Daniel has changed PrimarySlider component completely to something different for new version of design. However, that new design component slider was already existing in PrimarySliderV2 component. So I have reverted his changes back on PrimarySlider component back to as it was. I have also let him know about this. He will change it back to PrimarySliderV2.


This pr is now fully ready to be merged into main. @marvinmarnold
